src/models/mods.rs

```diff
@@ -38,10 +42,6 @@ pub struct Mod {
     pub categories: Vec<String>,
     /// A list of ids for versions of the mod.
     pub versions: Vec<VersionId>,
-
-    /// The latest version of the mod.
-    pub latest_version: Version,
```

Comment:
Does it need to be in the struct though? If an API client requests a mod they would be getting the version list anyways. I think a better approach is to make a struct with a "minified" version of the mod with less information.

Reply:
The latest version doesn't have to be in the struct itself, it could just be a `VersionId` instead of a full `Version`.  It would add a second request for most uses of the API, though: search listings show the latest supported MC version, mod pages show the last released files, etc.  When a client requests a mod, it only gets a list of the `VersionId`s of every version (which we may want to limit to the last 10 or so versions).  Finding the latest version if it isn't specified requires making a request for every version of the mod to check the info for each one.